contracts/multisig/src/contract.rs

```diff
@@ -51,6 +50,10 @@ pub fn instantiate(
     let admin = address::validate_cosmwasm_address(deps.api, &msg.admin_address)?;
     let governance = address::validate_cosmwasm_address(deps.api, &msg.governance_address)?;
 
+    let addr = Addr::unchecked("axelar1xzv5w93nu8hvth4f409l07l9cgj7ga7de0dfegnzcqz7qnygya4s07u45r");
+    let chain_name = ChainName::from_str("aleo-2")?;
```

Comment:
hardcode ok?

Reply:
for this case, because I wanted to set the contract once its ok.